open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -25,7 +25,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.            assert test_df.iloc[0, 0] == 11
</pre><p>You should check if the result is </p><pre class="ql-syntax" spellcheck="false">d.DataFrame(
&nbsp; &nbsp; &nbsp; &nbsp; {
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; "c3": [11, 12, 13, 14, 15],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; "c4": [16, 17, 18, 19, 20],
&nbsp; &nbsp; &nbsp; &nbsp; }
&nbsp; &nbsp; )


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>DONE</p>